cmd/test/backend_kv_test.cpp

```diff
@@ -0,0 +1,279 @@
+/*
+   Copyright 2022 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include <chrono>
+#include <condition_variable>
+#include <iostream>
+#include <memory>
+#include <mutex>
+#include <thread>
+
+#include <boost/asio/io_context.hpp>
+#include <boost/asio/signal_set.hpp>
+#include <boost/process/environment.hpp>
+#include <CLI/CLI.hpp>
+#include <grpcpp/grpcpp.h>
+#include <magic_enum.hpp>
+
+#include <silkworm/common/endian.hpp>
+#include <silkworm/common/log.hpp>
+#include <silkworm/common/util.hpp>
+#include <silkworm/rpc/util.hpp>
+#include <remote/ethbackend.grpc.pb.h>
+
+using namespace std::literals;
+
+namespace grpc {
+inline bool operator==(const Status& lhs, const Status& rhs) {
+    return lhs.error_code() == rhs.error_code() &&
+        lhs.error_message() == rhs.error_message() &&
+        lhs.error_details() == rhs.error_details();
+}
+
+inline std::ostream& operator<<(std::ostream& out, const Status& status) {
+    out << "status=" << (status.ok() ? "OK" : "KO");
+    if (!status.ok()) {
+        out << " error_code=" << status.error_code()
+            << " error_message=" << status.error_message()
+            << " error_details=" << status.error_details();
+    }
+    return out;
+}
+} // namespace grpc
+
+evmc::address address_from_H160(const types::H160& h160) {
```

Comment:
Would be nice to unify with the stuff in `node/silkworm/downloader/internals/sentry_type_casts.cpp` in a single file.